### ISSUE TITLE
Tournament Organizer Tab empty

### ISSUE THREAD

**Issue by LukasWi** [raphi011/scores#93]
e.g. Tournament ID 23530

**Comment by raphi011** [comment]
Yep this is a placeholder since it's not implemented yet .. if I can't get to implement it this week i'll remove the tab until it's done!
